java/ql/src/Security/CWE/CWE-352/SpringCSRFProtection.qhelp

```diff
@@ -30,9 +30,9 @@ OWASP:
 </li>
 <li>
 Spring Security Reference:
-<a href="https://docs.spring.io/spring-security/site/docs/current/reference/html5/#servlet-csrf">
+<a href="https://docs.spring.io/spring-security/reference/servlet/exploits/csrf.html#servlet-csrf">
```

Comment:
The URL for the page works fine, but the specified anchor doesn't seem to exist. Is there a subsection that we particularly want to take users to? `#disable-csrf` maybe?

Reply:
The original intention seemed to be to just link to the general Spring CSRF servlet page, not to a specific subsection, so I simply updated the URL to what appears to be the equivalent of the originally intended link. 
The Spring site gave me the `#servlet-csrf` anchor with the new URL 🤷, but I've removed it and also updated the page name in [335c597](https://github.com/github/codeql/pull/17795/commits/335c59792c99206e9fa8da86e46fac6bc96e2a1d).